webpack.config.js

```diff
@@ -48,6 +48,18 @@ module.exports = (shell = {}) => {
           'scheduler/tracing': 'scheduler/tracing-profiling',
         }),
         'safe-buffer': path.resolve(__dirname, 'node_modules/safe-buffer'),
+        '@emotion/serialize': path.resolve(
+          __dirname,
+          'node_modules/@emotion/serialize',
+        ),
+        '@emotion/utils': path.resolve(
+          __dirname,
+          'node_modules/@emotion/utils',
+        ),
+        '@emotion/sheet': path.resolve(
+          __dirname,
+          'node_modules/@emotion/sheet',
+        ),
```

Comment:
Why are these aliases necessary?

Reply:
they're to deal with some duplicate dependencies - will be removed again once the yarn PRs are merged